airbyte-webapp/src/views/Connection/CatalogTree/CatalogSection.tsx

```diff
@@ -153,7 +151,7 @@ const CatalogSectionInner: React.FC<TreeViewRowProps> = ({
         <StreamHeader
           stream={streamNode}
           destNamespace={destNamespace}
-          destName={prefix + streamNode.stream.name}
+          destName={prefix + streamNode.stream?.name}
```

Comment:
This could lead to prefix + 'undefined' ?

Reply:
The types say this can be undefined but I didn't encounter this in my testing. Without digging deep into the Java I'm not sure when this could occur.

Reply:
Would `${prefix}${streamNode.stream?.name ?? ''}` be safer?

Reply:
Did a `?? ""`